trackma/ui/cli.py

```diff
@@ -990,19 +990,20 @@ def _make_list_json(self, showlist):
                 "title": title_str,
                 "current_episode": episode_str_current,
                 "total_episodes": episode_str_last,
-                "score": score
+                "score": score,
+                "airing_status": False,
+                "behind_status": False
             }
 
-            # Color title according to status
+            # Check if show is airing and if user is behind
             if show['status'] == utils.Status.AIRING:
                 estimate = utils.estimate_aired_episodes(show)
                 if estimate and show['my_progress'] < estimate:
                     # User is behind the (estimated) aired episode
-                    j["color"] = _COLOR_BEHIND
+                    j["airing_status"] = True;
+                    j["behind_status"] = True;
```

Comment:
Still not too fond of this boolean (and the `status` in the field name). why not report the estimated last aired episode as-is?

Reply:
Hmm, this was as per 9cfc7d069dfe1e8daa8f703cd636c9c4dcb084ed as requested by myself in #659.
You are suggesting to print the last aired episode, and let scripts calculate if the user is caught up or behind?
Wouldn't it be confusing when "current_episode" exists?

Reply:
> Hmm, this was as per [9cfc7d0](https://github.com/z411/trackma/commit/9cfc7d069dfe1e8daa8f703cd636c9c4dcb084ed) as requested by myself in #659.

Not sure how this is relevant since that code is specifically about the CLI representation in the table for the user, not for tools.

> You are suggesting to print the last aired episode, and let scripts calculate if the user is caught up or behind?

If that is a property you (or other tools) are interested in, yes, exactly. It gives them more information to work with, such as being able to calculate *how far* they are behind, while still carrying the information of whether they are behind at all. As for naming, `estimated_aired_episode` sounds good to me, but it should only be included for airing shows (and can be omitted otherwise).

Reply:
Just to confirm, this is your proposed layout, correct:
```json
{"title": "Bungou Stray Dogs 4th Season", "current_episode": "2", "total_episodes": "13", "score": "0", "status": "Ongoing", "estimated_aired_episode": 2}
{"title": "Flying Witch", "current_episode": "9", "total_episodes": "12", "score": "0", "status": "Finished"}
```
Perhaps the "status" is redundant if estimated exists, but I do think the value is useful.

If have tested on my end the new format, Let's say the previous status format looked nicer in awk, but either way works.

Reply:
If you think status shouldn't be printed at all, I can simply remove line 988.

Reply:
Works for me, but I'm also not sure what you as the consumer of this API need here. If what you have is sufficient, then I'm fine with either. I'm just making sure that what is being added is reasonably implemented.

Reply:
For me it works like this. I have slightly more info than `list` does.
I will update the description.

Reply:
@justchokingaround  and @anma-dev are the only people I know that may benefit from this json format.

Reply:
@FichteFoll I also added some code to print covers image urls, as my script adl can display images This way, it should be provider agnostic, as my previous implementation was using anilist only, which may cause issues on mal trackma output for example.